schedule/qam/15/qam-gfxdrivers.yaml

```diff
@@ -25,5 +25,6 @@ schedule:
 - console/consoletest_finish
 - locale/keymap_or_locale_x11
 - x11/glxgears
-- shutdown/cleanup_before_shutdown
```

Comment:
is this intentional?

Reply:
Yes, it is.

Reply:
Added explanation to particular  commit message.

Reply:
Module breaks shutdown on the the bare metal system with gnome
installed. This module is usefull for qemu backends only, which is not
our case.